cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -57,16 +57,26 @@ public class InteractiveConfig {
     private Level logLevel;
 
     public static InteractiveConfig getDefault() {
+        String controllerURI = "localhost:9090";
+        boolean tlsEnabled = false;
+        //Default tls based configurations for pravega cli
+        if (System.getenv("PRAVEGA_CONTROLLER_URI") != null) {
+            controllerURI = System.getenv("PRAVEGA_CONTROLLER_URI");
```

Comment:
you are calling `System.getenv("PRAVEGA_CONTROLLER_URI")` twice, better to use a local variable instead.

Reply:
Incorporated as suggested